src/openapi/.redocly.yaml

```diff
@@ -0,0 +1,31 @@
+# See https://docs.redoc.ly/cli/configuration/ for more information.
+
+organization: adobe
+extends:
+    - minimal
+apis:
+  Commerce(Admin)@2.4.5:
+    root: openapi/admin-schema-2.4.5.yaml
```

Comment:
should all of these `root` paths be updated too?

Reply:
done, strange that the old links didn't break anything. i'm guessing that since it is a monorepo and the schemas are selected individually, it leaves less room for error.